controllers/eventing/cloudeventsource_controller.go

```diff
@@ -13,16 +13,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
+//
+//nolint:dupl
```

Comment:
I suppose the linter still complains despite our best efforts to reduce duplication?

Reply:
Yes